frontend/src/services/objectService.ts

```diff
@@ -124,12 +124,15 @@ export default {
   /**
    * @function updateObject
    * Update the object record (will add new version)
+   * @param {String} objectId The id for the object
+   * @param {Any} object Object to be created
```

Comment:
IIRC these types should be lowercase - we don't use the boxed string type, and any in caps doesn't exist.